src/index.ejs

```diff
@@ -8,7 +8,7 @@
     frame-src:
       - https://auspice.planx-pla.net and https://auspice.pandemicresponsecommons.org for COVID-19 Auspice app in iframe
     -->
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://login.bionimbus.org https://wayf.incommonfederation.org; child-src blob:; img-src 'self' https://opendata.datacommons.io https://static.planx-pla.net data: https://*.s3.amazonaws.com https://www.google-analytics.com; script-src 'self' 'unsafe-eval' https://www.google-analytics.com; worker-src 'self' blob:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; object-src 'none'; font-src 'self' data: https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self' https://login.bionimbus.org https://*.s3.amazonaws.com https://wayf.incommonfederation.org https://*.mapbox.com https://opendata.datacommons.io https://static.planx-pla.net <%= htmlWebpackPlugin.options.connect_src %>; frame-src <%= htmlWebpackPlugin.options.connect_src %> 'self' https://auspice.planx-pla.net https://auspice.pandemicresponsecommons.org;;">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://login.bionimbus.org https://wayf.incommonfederation.org; child-src blob:; img-src 'self' https://opendata.datacommons.io https://static.planx-pla.net data: https://*.s3.amazonaws.com https://www.google-analytics.com; script-src 'self' 'unsafe-eval' https://www.google-analytics.com <%= htmlWebpackPlugin.options.script_src %>; worker-src 'self' blob:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; object-src 'none'; font-src 'self' data: https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self' https://login.bionimbus.org https://*.s3.amazonaws.com https://wayf.incommonfederation.org https://*.mapbox.com https://opendata.datacommons.io https://static.planx-pla.net <%= htmlWebpackPlugin.options.connect_src %>; frame-src <%= htmlWebpackPlugin.options.connect_src %> 'self' https://auspice.planx-pla.net https://auspice.pandemicresponsecommons.org;;">
```

Comment:
i have doubts with it 😕 since `htmlWebpackPlugin` won't do anthing for `/dev.html`, so you have to end up with hard coding it in there. And I don't think that is what we want

Reply:
oh so you want add `htmlWebpackPlugin` support into `dev.html` as well?

Reply:
i guess we don't need to fire pagechange events from local dev
i was also just asking you here, i don't know the answer 😆

Reply:
it does make the life of debugging this thing easier, but that is all i think 🤷‍♂️